lib/pbench/test/unit/server/test_user_management_cli.py

```diff
@@ -0,0 +1,236 @@
+import datetime
+
+import pytest
+from click.testing import CliRunner
+
+from pbench.cli.server.user_management import (
+    USER_LIST_HEADER_ROW,
+    USER_LIST_ROW_FORMAT,
+    user_create,
+    user_delete,
+    user_list,
+    user_update,
+)
```

Comment:
Are we at the point where `import pbench.cli.server.user_management` would be better?  :-D

I think there's only one symbol that we're _not_ importing...and, given that this module exists to test that module, pulling the whole thing in seems appropriate....  But, don't make this change unless you have something else to change, too.

Reply:
I need a rebase so I will address it